benchmarks/conftest.py

```
@@ -0,0 +1,134 @@
+
+def get_rss_mb() -> float:
+    """Return current RSS in MiB (Linux/macOS)."""
+    try:
+        import resource
+
+        # ru_maxrss is in KiB on Linux, bytes on macOS
+        rss = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
+        if platform.system() == "Darwin":
+            return rss / (1024 * 1024)
+        return rss / 1024
+    except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)